QgisModelBaker/libqgsprojectgen/dataobjects/relations.py

```diff
@@ -10,6 +10,7 @@ def __init__(self):
         self.referencing_field = None
         self.referenced_field = None
         self.name = None
+        self.strength = 0
```

Comment:
Is that a "magic number"? what do the values mean?

Reply:
No. It's the value of `QgsRelation.Association` but I agree it can be made nicer using `QgsRelation.Association` so I made another commit.